dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/Admin/AddUserImportTests.cs

```diff
@@ -0,0 +1,181 @@
+using System.Text;
+using Microsoft.EntityFrameworkCore;
+
+namespace TeacherIdentity.AuthServer.Tests.EndpointTests.Admin;
+
+public class AddUserImportTests : TestBase
```

Comment:
As tests here depend on mocks need to ensure they don't run concurrently with other tests which may use the same mocks:
```suggestion
[Collection(nameof(DisableParallelization))]  // Relies on mocks
public class AddUserImportTests : TestBase
```

Reply:
Done